app/src/main/res/layout/walkthrough_activity.xml

```diff
@@ -42,6 +42,21 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+      android:id="@+id/walkthrough_activity_topic_header_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="16dp"
```

Comment:
we can use resources from dimens.xml

Reply:
So, we are talking about margins, right?

Reply:
yes, you can check https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/dimens.xml